modules/ROOT/pages/_partials/commons/common-p2psync-websocket-using-active.adoc

```diff
@@ -18,10 +18,10 @@ include::{root-partials}block-caveats.adoc[tags=code-disclaimer]
 
 [#introduction]
 == Introduction
-This content provides sample code and configuration examples covering the implementation of {glos-term-peer-to-peer-sync} over websockets.
-Specifically it covers the implementation of an {glos-term-active-peer}.
+This content provides sample code and configuration examples covering the implementation of {glos-term-peer-to-peer-sync} over WebSockets.
+Specifically it covers the implementation of a {glos-term-active-peer}.
```

Comment:
CAREFUL: https://docs.couchbase.com/couchbase-lite/current/android/p2psync-websocket-using-active.html#introduction the term is "Active Peer", so "an" was correct.

(This is one of the reasons that I was complaining about confusingly named fragments for keywords...)